### PR TITLE
Fix member remover

### PR DESCRIPTION
**Pull request by pakrym**
Run it after all the types were generated because strange things happen if some types are not found.

Fixes: https://github.com/Azure/autorest.csharp/issues/601

### CODE REVIEW

File: src/AutoRest.CSharp.V3/AutoRest/Plugins/GeneratedCodeWorkspace.cs [hunk 1]
```diff
@@ -70,6 +65,16 @@ public async IAsyncEnumerable<(string Name, string Text)> GetGeneratedFilesAsync
 
         private static async Task<Document> ProcessDocument(Document document)
         {
+            var compilation = document.Project.GetCompilationAsync().GetAwaiter().GetResult();
```
Comment (heaths) [inline src/AutoRest.CSharp.V3/AutoRest/Plugins/GeneratedCodeWorkspace.cs:68]: Now that you're in an async method, might as well use `await` for cleaner code.

File: test/TestProjects/TypeSchemaMapping/Generated/Models/CustomizedModel.cs [hunk 2]
```diff
@@ -12,16 +12,5 @@ namespace CustomNamespace
     /// <summary> The Model. </summary>
     internal partial class CustomizedModel
     {
-
-        /// <summary> Initializes a new instance of CustomizedModel. </summary>
-        /// <param name="propertyRenamedAndTypeChanged"> . </param>
-        /// <param name="customizedFancyField"> Fruit. </param>
-        /// <param name="daysOfWeek"> Day of week. </param>
-        internal CustomizedModel(int? propertyRenamedAndTypeChanged, CustomFruitEnum customizedFancyField, CustomDaysOfWeek daysOfWeek)
```
Comment (heaths) [inline test/TestProjects/TypeSchemaMapping/Generated/Models/CustomizedModel.cs:20]: I wasn't expecting this to be remove.d Isn't this the serialization ctor you need to deserialize?

Comment (pakrym) [reply test/TestProjects/TypeSchemaMapping/Generated/Models/CustomizedModel.cs:20]: leftovers of testing

File: src/AutoRest.CSharp.V3/AutoRest/Plugins/GeneratedCodeWorkspace.cs (hunk 1 above)
Comment (pakrym) [reply src/AutoRest.CSharp.V3/AutoRest/Plugins/GeneratedCodeWorkspace.cs]: Good point.

**Pull request merged**